app/classifier/index.cjsx

```diff
@@ -49,10 +75,15 @@ Classifier = React.createClass
       @loadSubject subject
     if nextProps.classification isnt @props.classification
       @prepareToClassify nextProps.classification
+    @context.geordi?.remember subjectID: @props.subject?.id
+    @reCheckIfInterventionNeeded()
```

Comment:
Yeah I saw your other comment, will see about moving it out of context. To be honest I've been having some problems keeping a single, constant singleton class in existence without having it in context. I will try the defaultProps approach you suggested.